DQMOffline/L1Trigger/interface/L1TStage2CaloLayer2Offline.h

```diff
@@ -111,6 +111,9 @@ class L1TStage2CaloLayer2Offline: public DQMEDAnalyzer {
   // Histograms
   MonitorElement* h_nVertex_;
 
+  // energy sums control plots
+  std::map<std::string, MonitorElement*> h_energySum_controlPlots_;
```

Comment:
Hi @kreczko - please change the std::string here and below into a set of enums to avoid endless string comparisons in the event loop.